cpp/tests/io/csv_test.cpp

```diff
@@ -1531,4 +1531,25 @@ TEST_F(CsvReaderTest, DurationsWithWriter)
   CUDF_TEST_EXPECT_TABLES_EQUIVALENT(input_table, result_table);
 }
 
+TEST_F(CsvReaderTest, ReadMaxNumericValue)
```

Comment:
Some cases don't have test coverage:
- outside of uint64 range, positive
- outside of int64t range, negative
- outside of int64t range but abs(val) is smaller than uint64t max
- all of the above with preceding zeroes

Basically, we need a test for each branch in the `get_counter_address `function, so there's probably more.